taming/data/cub_dataloader.py

```diff
@@ -0,0 +1,43 @@
+from torch.utils.data import Dataset
+from taming.data.base import ImagePaths
```

Comment:
Looking at this again, I don not see a good reason for reason for creating a duplicate of the Custom dataset objects. either use inheritance or just use same objects